mypy/config_parser.py

```diff
@@ -278,6 +280,13 @@ def parse_section(prefix: str, template: Options,
                 if 'follow_imports' not in results:
                     results['follow_imports'] = 'error'
         results[options_key] = v
+        if key in ('files', 'packages', 'modules'):
+            if all(('packages' in results.keys() or 'modules' in results.keys(),
+                    results.get('files'))):
```

Comment:
Style nit: This was a bit difficult to understand. I think it would better to use `and` instead of `all()`, and you can just use `'packages' in results`, without using `keys()`, I think.

Reply:
That's fair.  I started with something that generated a list and tested it and didn't refactor it out as far as I should have.  Fixed.